data/en/bots/330332878070218754.md

```diff
@@ -0,0 +1,55 @@
+---
+avatar: >-
+  https://cdn.discordapp.com/avatars/488805846499721216/9fdc16f903d7c2bdf9da3e9646095878.png?size=2048
+cover: null
+custom_path: !<tag:yaml.org,2002:js/undefined> ''
+description: ' Nydea  » Nydea have more than ten commands modules, it have all in one. The bot that you look in your dreams, it''s here.'
+github: null
+images: []
+link: '330332878070218754'
+nsfw: false
+pagename: Nydea
+prefix: ny! or custom
+support: 'https://discord.gg/B4E2jF3'
+---
+## Nydea » a discord powerful bot
+
+---
+
+### › ¿What is nydea? 
+
+**Nydea is a discord bot that can power-up your server, nydea have all the features that are you looking for, with super-special things that can't do a lot of bots in this time. It was developed in commando, is was harder but we developed perfectly nydea, with dedication and good feel.**
+
+---
+
+### Special Features
+
+**That i mentioned above, i will tell you what are the "SUPER-SPECIAL" things that i mentioned. It is...**
+
+> Command auto-cancel after 30 seconds.
+
+> Command cancel saying `cancel`.
+
+> Double prefix `ny!help` and `@Nydea#8387 help`.
+
+> Auto-command adding with description.
+
+> DM work commands.
+
+> Commands needing mention an user, you can use instead `ID | USERNAME | MENTION`.
+
+> Multiple command aliases, watch aliases using `ny!alias <command>`
+
+> Basic commands like `help, ping, prefix` with a awesome function.
+
+> Unknow command **response**.
```

Comment:
Is that really a "feature"?
Imo should no bot really have such a thing...

Reply:
"Awesome ping"
"Awesome prefix"
"Awesome help"

Reply:
Awesome!